src/js/views/maps/feature-tiles-layer.js

```diff
@@ -0,0 +1,236 @@
+/*jslint nomen: true */
+/*globals define */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+  var _ = require('lib/lodash');
+  var Backbone = require('backbone');
+  var L = require('lib/leaflet/leaflet.tilejson');
+  var Promise = require('lib/bluebird');
+
+  var api = require('api');
+  var util = require('util');
+
+  // We reuse the cartodb layer templates
+  var infoTemplate = require('text!templates/cartodb-info.html');
```

Comment:
Minor, but can you rename these files so it's clear they're not cartodb-only? I can see a situation where we make a change that's only supposed to affect cartodb views and don't know to check other scenarios.